config/src/config/key_manager_config.rs

```diff
@@ -32,6 +34,7 @@ impl Default for KeyManagerConfig {
             secure_backend: SecureBackend::InMemoryStorage,
             sleep_period_secs: DEFAULT_SLEEP_PERIOD_SECS,
             txn_expiration_secs: DEFAULT_TXN_EXPIRATION_SECS,
+            chain_id: ChainId::mainnet(),
```

Comment:
I'd be concerned with a default that connects to the mainnet chain.  I'll let @JoshLind make the call on that one though.

Reply:
I don't have strong feelings, but I think this should probably match whatever default we're using in the actual validator/node configs.